CONTRIBUTING.md

```diff
@@ -126,11 +126,9 @@ spice run
 sudo apt update
 sudo apt install build-essential curl openssl libssl-dev pkg-config protobuf-compiler cmake
 
-# Detect the machine's architecture
-ARCH=$(uname -m)
-
-# Map uname output to Go's naming convention
-case "$ARCH" in
+# Install Go
+ARCH=$(uname -m) # Detect the machine's architecture
```

Comment:
`uname -m` returns `arm64` for me on macOS 🤔

Reply:
```
sergey@warm:~/sergey-spiceai$ uname -m
aarch64
```

Reply:
It looks for Debian (Ubuntu) it should be `aarch64`: https://wiki.debian.org/ArchitectureSpecificsMemo

Reply:
Found the answer here https://stackoverflow.com/questions/31851611/differences-between-arm64-and-aarch64 :

>It seems that ARM64 was created by Apple and AARCH64 by the others, most notably GNU/GCC guys.

>After some googling I found this link:

>[The LLVM 64-bit ARM64/AArch64 Back-Ends Have Merged](https://www.phoronix.com/news/MTY5ODk)

>So it makes sense, iPad calls itself ARM64, as Apple is using LLVM, and Edge uses AARCH64, as Android is using GNU GCC toolchain.

Reply:
TIL! Given this is for the Linux section, I think we can leave it